app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb

```diff
@@ -206,4 +204,12 @@ def self.raw_connect(region, endpoint, access_key, secret_key)
 
     Aws.const_get(:S3)::Resource.new(options).client
   end
+
+  def remove_object(bucket_name, object_name)
+    connect.delete_object({:bucket => bucket_name, :key => object_name})
```

Comment:
If the only argument is a hash you don't need the `{}` around it: `connect.delete_object(:bucket => bucket_name, :key => object_name)`

Reply:
done